web/beacon-app/src/features/organization/components/OrganizationTable.tsx

```diff
@@ -45,7 +45,19 @@ export default function OrganizationsTable() {
               { Header: 'Organization Name', accessor: 'name' },
               { Header: 'Organization Role', accessor: 'role' },
               { Header: 'Projects', accessor: 'projects' },
-              { Header: 'Date Created', accessor: 'created' },
+              {
+                Header: 'Date Created',
+                accessor: (date: any) => {
+                  return Intl.DateTimeFormat(undefined, {
+                    localeMatcher: 'best fit',
```

Comment:
After our discussion about choosing the locale, I read that `best fit` is a better choice than `lookup`.